src/documentation/pages/components/button/examples/_danger-lumen-centurylink.vue

```diff
@@ -0,0 +1,48 @@
+<template lang="pug">
+  <ComponentExample title="Danger" id="danger-lumen-centurylink" :tabs="exampleTabs">
+    p.-text(slot="example-description")
+      | For special cases, such as styling a destructive action in an application (e.g. Delete Account),
+      | a Danger button may be used.
+    div(slot="example")
```

Comment:
we should use Wrapper here instead of a div

Reply:
@evansito I thought we could use div instead of wrapper when we have similar multiple elements and wrapper when we have wrap different elements. Please correct me if i'm wrong.

Reply:
Wrapper component should be used when there are multiple non-reactive items and there isn't a need for an extra div (it doesn't matter if the elements are similar or different e.g. three divs or one div, one p and one img)